adapters/repos/db/helpers/helpers.go

```diff
@@ -25,6 +28,49 @@ var (
 	DocIDBucket                = []byte("doc_ids")
 )
 
+func MakePropertyPrefix(property string, propIds *tracker.JsonPropertyIdTracker) ([]byte, error) {
+	propid, err := propIds.GetIdForProperty(string(property))
+	if err != nil {
+		fmt.Printf("property '%s' not found in propIds, creating", property)
+	}
+	propid_bytes := make([]byte, 8)
+	binary.LittleEndian.PutUint64(propid_bytes, propid)
+	return propid_bytes, nil
+}
+
+func MakePropertyKey(propPrefix []byte, key []byte) []byte {
+	if len(propPrefix) == 0 {
+		return nil
+	}
+
+	t := key[:]
+	val := append(t, propPrefix...)
+
+	return val
+}
+
+func MatchesPropertyKeyPrefix(propName []byte, key []byte) bool {
+	if len(propName) == 0 {
+		fmt.Println(fmt.Errorf("Empty property name in MatchesPropertyKeyPrefix, this is almost certainly wrong"))
+		return false
+	}
+
+	return bytes.HasSuffix(key, propName)
+}
+
+func UnMakePropertyKey(propName []byte, key []byte) []byte {
+	if len(propName) == 0 {
+		fmt.Println(fmt.Errorf("Empty property name in UnMakePropertyKey, this is almost certainly wrong"))
```

Comment:
why is here `fmt.Println` together with error? isn't it better to use a logger here?

Reply:
We should use the logger here, as there is going to be a push soon to aggregate+pipe all logs into external services